.gitlab/lassen-jobs.yml

```diff
@@ -52,7 +52,7 @@ xl_16_1_1_12_gcc_8_3_1:
 
 clang_11_cuda:
```

Comment:
Along the same lines, and to make things consistent, perhaps this should be `clang_11_0_0_cuda_11`?

Reply:
This will be resolved after merge in separate PR!